api/server.ts

```
@@ -1,12 +1,20 @@
+import z from "zod";
 import { initializeMcpApiHandler } from "../lib/mcp-api-handler";
 import registerTools from "./tools";
 
+const apiKeySchema = z.object({
+  apiKey: z.string().min(1, "API key is required"),
+});
+
 const handler = initializeMcpApiHandler(
   (server, apiKey) => {
     // Register Meeting BaaS SDK tools with the provided API key
     server = registerTools(server, apiKey);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)